doc/source/_static/custom.css

```diff
@@ -1,16 +1,164 @@
 .sphinx-codeautolink-a{
-    border-bottom-color: rgb(0, 0, 0);
-    border-bottom-style: dotted;
-    border-bottom-width: 1px;
+  border-bottom-color: rgb(0, 0, 0);
+  border-bottom-style: dotted;
+  border-bottom-width: 1px;
 }
 .sphinx-codeautolink-a:hover{
-    color: rgb(255, 139, 139);
+  color: rgb(255, 139, 139);
 }
 
 .supportbutton a {
- color: var(--pst-color-sidebar-link-active);
+color: var(--pst-color-sidebar-link-active);
 }
 
 .navbar-brand {
-  display: inline-block !important;
+display: inline-block !important;
+}
+
+/* -------------------- THEME OVERRIDES -------------------- */
+
+html[data-theme="light"] {
+  --pst-color-primary: rgb(11 117 145);
+  --pst-color-secondary: rgb(238 144 64);
+}
+
+html[data-theme="dark"] {
+  --pst-color-primary: rgb(0 192 191);
+  --pst-color-secondary: rgb(238 144 64);
+}
+
+.bd-content {
+  flex-grow: 1;
+  max-width: 60em;
```

Comment:
maybe we can try 70em? it feels a bit too narrow still, especially in the example gallery with the sidebar taking extra space too

Using 100% as recommended in https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/layout.html#horizontal-spacing would also be great, but then we'd need to restructure the page to avoid having text that is too wide.